### ISSUE TITLE
Save & Close Button Fails After Module Text Update

### ISSUE THREAD

**Issue by dguyton** [joomla/joomla-cms#25643]
### Steps to reproduce the issue
1. Open a menu item for editing
2. Click on Module Assignment tab
3. Hide unassigned modules
4. Hide unpublished modules
5. Select a module with editable text
6. Make edits to the text and click the "Save & Close" button

### Expected result
Text edits are saved

### Actual result
Text edits are not saved (note it works if you click Save instead, and then Close)

### System information (as much as possible)

3.9.10
Setting 	Value
 
PHP Built On 	Linux us188.siteground.us 3.12.18-clouder0 #1 SMP Wed Mar 28 14:22:08 EEST 2018 x86_64
Database Type 	mysql
Database Version 	5.6.40-84.0-log
Database Collation 	utf8_general_ci
Database Connection Collation 	utf8mb4_general_ci
PHP Version 	7.1.30
Web Server 	Apache
WebServer to PHP Interface 	cgi-fcgi
Joomla! Version 	Joomla! 3.9.10 Stable [ Amani ] 10-July-2019 15:57 GMT
Joomla! Platform Version 	Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent 	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) Gecko/20100101 Firefox/67.0 

### Additional comments

Tested on both Firefox and Chrome browsers (same result)![screen shot 2019-07-19 at 13 33 58](https://issues.joomla.org/uploads/1/dc6624b60f71b1e7604379018a99a9e4.png)![screen shot 2019-07-19 at 13 34 00](https://issues.joomla.org/uploads/1/17cc8213874555d87bf63d54bb151370.png)

**Comment by ghost** [comment]
Can't confirm. Save&Close works as expected.

Setting | Value
-- | --
PHP Built On | Linux lamp126.cloudaccess.net 3.10.0-962.3.2.lve1.5.24.5.el6h.x86_64 #1 SMP Thu Nov 22 10:07:47 EST 2018 x86_64
Database Type | mysql
Database Version | 5.7.23-cll-lve
Database Collation | utf8_general_ci
Database Connection Collation | utf8mb4_general_ci
PHP Version | 7.3.7
Web Server | Apache
WebServer to PHP Interface | cgi-fcgi
Joomla! Version | Joomla! 3.9.10 Stable [ Amani ] 10-July-2019 15:57 GMT
Joomla! Platform Version | Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:68.0) Gecko/20100101 Firefox/68.0

**Comment by ghost** [comment]
Have you tried disabling 3rd-Party-Plugins and asked first on [Forum](https://forum.joomla.org/)?

**Comment by Quy** [comment]
Unable to reproduce.

**Comment by ghost** [comment]
closed as unconfirmed Report.

**Comment by dguyton** [comment]
> Have you tried disabling 3rd-Party-Plugins and asked first on [Forum](https://forum.joomla.org/)?

Thanks to all of you for verifying (even tho it's me.. not Joomla). I did not post in the forums, but I did search them first and couldn't find anything relevant. No doubt part of the issue is I'm new to Jooma. However, my understanding is the issue I'm running into is Joomla's default system of creating content (which I would think would not be impacted by plugins, since it's not a site issue from a user perspective).
